packages/zod-rpc/readme.md

```diff
@@ -0,0 +1,244 @@
+# zod-rpc
+
+## RPC calls
+
+### 1 Create a new RPC call that **requires** a response
+
+In any file, create a class with the following structure, note that the `validator` here would
+be located in a sibling file with the `.zod.js` extension. This is how we are able to strip validation
+for the production builds.
```

Comment:
Is it really _any_ file? Should we suggest an idea structure? In your example, the validator is under the `rpc` folder. Should we suggest this the way to go? Even if not, you can point to that as an example ("see for example `src/rpc/rpc-calls.js` and `src/rpc/validators.zod.js`"). You can even show an example of folder structure like:

```
src
| newRpc
| | new-rpc-calls.js
| | validators.jzod.js
```